src/main/docker/bin/run

```diff
@@ -3,7 +3,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $DIR/container-limits
 
-export JAVA_OPTS=$($DIR/java-options)
+#export JAVA_OPTS=$($DIR/java-options)
```

Comment:
Denne må vel bort hvis ikke får vi ikke noen JAVA_OPTS satt. Altså kommentar tegnet.

Reply:
File updated to be in sync with flange.